### PR TITLE
Add quickstart on environment variables

### LINKED ISSUE #306: Add a quickstart on using environment variables
Talk about injected values and manual values

### PR DESCRIPTION
**Pull request by AaronCrawfis**
Closes #306

### CODE REVIEW

File: docs/content/getting-started/quickstarts/quickstart-environment-variables/index.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+   rad deploy ./app.bicep
+   ```
+1. Port-forward the container to your machine:
+
+    ```bash
+    rad resource expose containers mycontainer -a myapp --port 5000
+    ```
+1. Visit [localhost:5000](http://localhost:5000) in your browser. You should see the following page, now showing injected environment variables:
+
+   <img src="screenshot-all.jpg" alt="Screenshot of the app printing all the environment variables" width=1000px />
```
Comment (Reshrahim) [inline docs/content/getting-started/quickstarts/quickstart-environment-variables/index.md:78]: screenshots do not load..maybe wrongpath
image

**Comment by Reshrahim** [comment]
@AaronCrawfis - looks like the workflow is changed? doesn't require approval from reviewers?

File: docs/content/getting-started/quickstarts/quickstart-environment-variables/index.md (hunk 1 above)
Comment (AaronCrawfis) [reply docs/content/getting-started/quickstarts/quickstart-environment-variables/index.md:78]: It appears to be working in the staging site. Are you getting this error when running locally?

**Comment by AaronCrawfis** [comment]
> @AaronCrawfis - looks like the workflow is changed? doesn't require approval from reviewers?

Good catch, our old regex started not matching at the 0.12 release. Just fixed it

File: docs/content/getting-started/quickstarts/quickstart-environment-variables/index.md (hunk 1 above)
Comment (Reshrahim) [reply docs/content/getting-started/quickstarts/quickstart-environment-variables/index.md:78]: Screenshot is from the staging site. Not sure y it isn't loading :(

Comment (AaronCrawfis) [reply docs/content/getting-started/quickstarts/quickstart-environment-variables/index.md:78]: Could you hit F12 and see if your browser console is throwing any errors?

Comment (Reshrahim) [reply docs/content/getting-started/quickstarts/quickstart-environment-variables/index.md:78]: 404 
image

Comment (AaronCrawfis) [reply docs/content/getting-started/quickstarts/quickstart-environment-variables/index.md:78]: Huh. I have absolutely no idea. It appears to be looking for the screenshots one level above where it should be. And it's working on my browser with the exact same URL of the quickstart. Which browser are you using?

Comment (Reshrahim) [reply docs/content/getting-started/quickstarts/quickstart-environment-variables/index.md:78]: Tried in edge, chrome. All same behavior. Let's merge and validate if the behavior exists

**Review by Reshrahim — APPROVED** [review]

**Pull request merged**